lib/kitto/hooks.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Kitto.Hooks do
+  @moduledoc """
+  Kitto Hooks enable an alternative to Jobs for feeding data into dashboards.
+  Hooks enable remote services to push data into Kitto using webhooks.
+
+  Just like jobs, hooks are loaded at runtime from the `hooks/` directory at
+  the root of the application. Hooks can be defined as follows:
+
+      use Kitto.Hooks.DSL
+      hook :hello do
+        {:ok, body, _} = read_body conn
+        broadcast! :hello, body |> Poison.decode!
+      end
+  """
+
+  use GenServer
+
+  def start_link do
```

Comment:
Starting it like https://github.com/kittoframework/kitto/blob/master/lib/kitto/runner.ex#L17
with a configurable name allows us to test it independently by spawning a new process (see: https://github.com/kittoframework/kitto/blob/master/test/runner_test.exs#L32) and then disposing it.